content/events/2018/tech-day-02-17.md

```diff
@@ -3,15 +3,8 @@ title: February Tech Day 2
 date: 2018-02-17
 ---
 
-{{< location 25 >}}
+{{% location building-25 %}}
 
 The day will run from 10:00am until 5:00pm, however feel free to come and go as you please.
 
-## Food
-Food on campus is scarce on weekends, so we recommend either ordering in food, or bringing something with you. There are a few corner-shops located nearby however. There's a [map of local food places](https://goo.gl/yYlfs5) to help you find nearby food.
-
-## Electronics
-As expected, you'll need to bring your robot to work on, in whatever state it may be. We also recommend bringing at least 1 laptop with you to work on. Laptops will be provided for you to use.
-
-## Supervision
-Team members must be accompanied by any adult who is able to take responsibility for all of them. (this would ideally be a teacher, but it can be a parent if all the other parents agree).
+{{% techday-details %}}
```

Comment:
Ah, sorry, I was only expecting the "Supervision" content to be deduplicated as it'll always be the same. Will the food & electricals sections not vary by tech day/location?

I can see some value in extracting each of them, though maybe we should have each separately?:
```
{[% food-southampton %}}
{[% electronics-laptops-provided %}}
{{% supervision %]]
```
I'd also be happy with just duplicating the food & electronics bits.

Reply:
The food section will be the same all the time tech days are at Highfield / Boldrewood, the requirement on bringing a laptop will also be the same, getting logins is pretty difficult and unnecessary for things like this. Note the shortcode is only for tech days, not all events in general

Reply:
I appreciate that this is the case for now, my concern is that this bakes in the assumption that all tech-days will be on the University Campus and that the electronics situation will always be the same.
This feels very much like something which would be _really easy_ to miss when reviewing the addition of a tech day, in a way that forcing the separation of the specifications (either via separate clearly named shortcodes or plain inline content) avoids.

Essentially I see this as a loaded foot-gun which could be trivially avoided.